src/EPR.Calculator.Frontend/Views/CalculationRunName/Confirmation.cshtml

```diff
@@ -0,0 +1,15 @@
+﻿﻿@{
+    Layout = "_Layout";
+    string calculationName = !string.IsNullOrWhiteSpace(ViewBag.CalculationName)
+                                ? ViewBag.CalculationName.ToString()
+                                : Context.Session.GetString("CalculationName") ?? "";
+
+    var model = new ConfirmationViewModel
+            {
+                Title = "You have started a calculation",
+                Body = calculationName,
+                NextText = " It will take up to 15 minutes for the calculation to finish. You will be able to view it in the list of calculations on the dashboard."
```

Comment:
Does this need to be in a model? This can be part of the content
- You have started a calculation
- It will take up to 15 minutes for the calculation to finish. You will be able to view it in the list of calculations on the dashboard.
-

Reply:
Please ignore this comment. I got this wrong.